<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RankingView</class>
 <widget class="QDialog" name="RankingView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ranking</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="vsLabel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>VS.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="QuitButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>380</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="team_ranking">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>141</width>
     <height>281</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="chrono_ranking">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>141</width>
     <height>281</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="vs_ranking">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>90</y>
     <width>141</width>
     <height>281</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="chronoLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>30</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Chrono.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="teamLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Team.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>-60</x>
     <y>-20</y>
     <width>621</width>
     <height>441</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="pixmap">
    <pixmap>ranking.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>vsLabel</zorder>
  <zorder>QuitButton</zorder>
  <zorder>team_ranking</zorder>
  <zorder>chrono_ranking</zorder>
  <zorder>vs_ranking</zorder>
  <zorder>chronoLabel</zorder>
  <zorder>teamLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
